sdkproject/Assets/Mapbox/Unity/Utilities/PolygonUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using Mapbox.VectorTile.Geometry;
+using Mapbox.VectorTile.Geometry.InteralClipperLib;
```

Comment:
Using `clipper` works for now, but be aware that this will break once our VT APIs support server side clipping (no ETA yet) and `vector-tile-cs` will drop that dependency.

Reply:
@BergWerkGIS What is `server side clipping` meant to do?

Reply:
yes, server side clipping.